### DISCUSSION TITLE
How to add custom javascipt for Matomo or Google Analytics?

### DISCUSSION THREAD

**Discussion by prasannakulkarni333** [getpelican/pelican#3056] [Q&A]
Hello, I have a blog live with pelican. I want to track hence want to put the tracking code in the <head> section for analytics tools - Matomo, GA4,GTM etc. 
How can I do that with pelican?

I believe I should be doing this in the base jinja theme so that I don't repeat it on all the pages.

Thanks.

**Comment by justinmayer** [comment]
Hi @prasannakulkarni333. The answer to your question is entirely theme-dependent. For an example of how this could be done, see the [default theme's `analytics.html` template](https://github.com/getpelican/pelican/blob/master/pelican/themes/notmyidea/templates/analytics.html).

**Reply by prasannakulkarni333** [reply]
OK I see. Thanks. 
I am using notmyidea, will check how I can change it.
